### PR TITLE
add the lgm 2016

### PR DESCRIPTION
**Pull request by aoloe**

### CODE REVIEW

File: _posts/2016-04-18-libre-graphics-meeeting.markdown [hunk 1]
```diff
@@ -0,0 +1,22 @@
+---
+layout: post
+title:  "Libre Graphics Meeting 2016
```
Comment (jancborchardt) [inline _posts/2016-04-18-libre-graphics-meeeting.markdown]: There seems to be a stray double quote here. :)

Comment (jancborchardt) [inline _posts/2016-04-18-libre-graphics-meeeting.markdown]: Or rather, the ending one is missing.

**Comment by aoloe** [comment]
i've added a "

**Comment by ei8fdb** [comment]
Okie dokie.

Merged!

**Pull request merged**